### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by Tirokk**
> [appinteractive](https://github.com/appinteractive) **Authored by [appinteractive](https://github.com/appinteractive)**
_<time datetime="2019-11-17T16:48:10Z" title="Sunday, November 17th 2019, 5:48:10 pm +01:00">Nov 17, 2019</time>_
_Closed <time datetime="2019-11-18T21:33:25Z" title="Monday, November 18th 2019, 10:33:25 pm +01:00">Nov 18, 2019</time>_
---

This pull request was initiated by Lokalise (user Mogge) at 2019-11-17 17:47:59

### CODE REVIEW

**Comment by Tirokk** [comment]
> [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-17T16:52:48Z" title="Sunday, November 17th 2019, 5:52:48 pm +01:00">Nov 17, 2019</time>_

---


This PR was triggered by Lokalize as indicated in the picture. To see this screen you have to click on the Download icon.

grafik

File: locale/fr.json [hunk 1]
```diff
@@ -0,0 +1,2122 @@
...
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
```
Comment (Tirokk) [inline locale/fr.json:16]: > [Elweyn](https://github.com/Elweyn) **Authored by [Elweyn](https://github.com/Elweyn)**
_<time datetime="2019-11-18T12:24:19Z" title="Monday, November 18th 2019, 1:24:19 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,2122 @@
+{
+    "login": {
+        "copy": {
+            "translation": "Si vous avez déjà un compte human-connection, connectez-vous ici.",
+            "notes": ""
+        },
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
```


"Votre courriel" seems a bit strange, I think it should be "Votre mail" or "Votre addresse mail"

File: locale/fr.json [hunk 2]
```diff
@@ -0,0 +1,2122 @@
...
+        "moreInfoHint": {
+            "translation": "à la page de présentation",
+            "notes": ""
+        },
+        "success": {
+            "translation": "Vous êtes connecté!",
+            "notes": ""
+        },
+        "failure": {
+            "translation": "Adresse e-mail ou mot de passe incorrect.",
```
Comment (Tirokk) [inline locale/fr.json:56]: > [Elweyn](https://github.com/Elweyn) **Authored by [Elweyn](https://github.com/Elweyn)**
_<time datetime="2019-11-18T12:25:16Z" title="Monday, November 18th 2019, 1:25:16 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,2122 @@
+{
+    "login": {
+        "copy": {
+            "translation": "Si vous avez déjà un compte human-connection, connectez-vous ici.",
+            "notes": ""
+        },
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
+            "notes": ""
+        },
+        "password": {
+            "translation": "Votre mot de passe",
+            "notes": ""
+        },
+        "moreInfo": {
+            "translation": "Qu'est-ce que Human Connection?",
+            "notes": ""
+        },
+        "hello": {
+            "translation": "Bonjour",
+            "notes": ""
+        },
+        "forgotPassword": {
+            "translation": "Mot de passe oublié?",
+            "notes": ""
+        },
+        "no-account": {
+            "translation": "Vous n'avez pas de compte?",
+            "notes": ""
+        },
+        "register": {
+            "translation": "S'inscrire",
+            "notes": ""
+        },
+        "moreInfoURL": {
+            "translation": "https:\/\/human-connection.org\/fr\/",
+            "notes": ""
+        },
+        "moreInfoHint": {
+            "translation": "à la page de présentation",
+            "notes": ""
+        },
+        "success": {
+            "translation": "Vous êtes connecté!",
+            "notes": ""
+        },
+        "failure": {
+            "translation": "Adresse e-mail ou mot de passe incorrect.",
```


Changed 'e-mail' to 'mail' on localise

File: locale/fr.json [hunk 3]
```diff
@@ -0,0 +1,2122 @@
...
+        "following": {
+            "translation": "Suivant",
+            "notes": ""
+        },
+        "shouted": {
+            "translation": "Recommandé",
+            "notes": ""
+        },
+        "commented": {
+            "translation": "Comentado",
```
Comment (Tirokk) [inline locale/fr.json:86]: > [Elweyn](https://github.com/Elweyn) **Authored by [Elweyn](https://github.com/Elweyn)**
_<time datetime="2019-11-18T12:27:53Z" title="Monday, November 18th 2019, 1:27:53 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,2122 @@
+{
+    "login": {
+        "copy": {
+            "translation": "Si vous avez déjà un compte human-connection, connectez-vous ici.",
+            "notes": ""
+        },
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
+            "notes": ""
+        },
+        "password": {
+            "translation": "Votre mot de passe",
+            "notes": ""
+        },
+        "moreInfo": {
+            "translation": "Qu'est-ce que Human Connection?",
+            "notes": ""
+        },
+        "hello": {
+            "translation": "Bonjour",
+            "notes": ""
+        },
+        "forgotPassword": {
+            "translation": "Mot de passe oublié?",
+            "notes": ""
+        },
+        "no-account": {
+            "translation": "Vous n'avez pas de compte?",
+            "notes": ""
+        },
+        "register": {
+            "translation": "S'inscrire",
+            "notes": ""
+        },
+        "moreInfoURL": {
+            "translation": "https:\/\/human-connection.org\/fr\/",
+            "notes": ""
+        },
+        "moreInfoHint": {
+            "translation": "à la page de présentation",
+            "notes": ""
+        },
+        "success": {
+            "translation": "Vous êtes connecté!",
+            "notes": ""
+        },
+        "failure": {
+            "translation": "Adresse e-mail ou mot de passe incorrect.",
+            "notes": ""
+        }
+    },
+    "profile": {
+        "name": {
+            "translation": "Mon profil",
+            "notes": ""
+        },
+        "memberSince": {
+            "translation": "Membre depuis",
+            "notes": ""
+        },
+        "follow": {
+            "translation": "Suivre",
+            "notes": ""
+        },
+        "followers": {
+            "translation": "Suiveurs",
+            "notes": ""
+        },
+        "following": {
+            "translation": "Suivant",
+            "notes": ""
+        },
+        "shouted": {
+            "translation": "Recommandé",
+            "notes": ""
+        },
+        "commented": {
+            "translation": "Comentado",
```


Need a new pull from localise, has been changed

**Comment by Tirokk** [comment]
> [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-18T20:33:04Z" title="Monday, November 18th 2019, 9:33:04 pm +01:00">Nov 18, 2019</time>_

---


@roschaefer this is fixed:
grafik

**Comment by Tirokk** [comment]
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-18T21:33:25Z" title="Monday, November 18th 2019, 10:33:25 pm +01:00">Nov 18, 2019</time>_

---


@Mogge @Elweyn alright, so I'm going to close this? We can trigger another PR from lokalise any time.

Funny by the way that @appinteractive is the PR creator.. :thinking:

**Comment by Tirokk** [comment]
> [Mogge](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-18T22:24:39Z" title="Monday, November 18th 2019, 11:24:39 pm +01:00">Nov 18, 2019</time>_

---


Yes, please close. We should establish an import from lokalise once a week.

**Review by Mogge — CHANGES_REQUESTED** [review]
This is weird. The folder should be `webapp/locales` not `locale/`.

**Pull request closed without merging**